index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="es" xml:lang="es">
+<head>
+  <title>Koombea - Training</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name = "keywords" content = "koombea, html, css, javascript" />
```

Comment:
@edwartexe avoid these spaces between the property and its value to improve code readability. Syntax is valid, but it is difficult to read.
```suggestion
  <meta name="keywords" content="koombea, html, css, javascript" />
```